crates/crates_io_database/src/schema.rs

```diff
@@ -538,6 +538,12 @@ diesel::table! {
         ///
         /// (Automatically generated by Diesel.)
         token_generated_at -> Nullable<Timestamptz>,
+        /// The `send_notifications` column of the `emails` table.
+        ///
+        /// Its SQL type is `Bool`.
+        ///
+        /// (Automatically generated by Diesel.)
```

Comment:
if you add a comment to the column in the migration then diesel will automatically use that for the Rust schema field comment too 😉